unstable/wlr-screencopy-unstable-v1.xml

```diff
@@ -175,5 +175,27 @@
         Destroys the frame. This request can be sent at any time by the client.
       </description>
     </request>
+
+    <request name="copy_with_damage" since="2">
+      <description summary="copy the frame when it's damaged">
+        Same as copy, except it waits until there is damage to copy.
+      </description>
+      <arg name="buffer" type="object" interface="wl_buffer"/>
+    </request>
+
+    <event name="damage" since="2">
+      <description summary="carries the coordinates of the damaged region">
+        This event is sent right before the ready event when copy_with_damage is
+        requested.
```

Comment:
This doesn't specify this is only sent once. Either we should make it mandatory to send it only once, either we should allow sending it multiple times, describing the fine-grained region.

Reply:
Yes, we could add fine-grained damage later by specifying that this may be called multiple times.